tailwind.purge.safelist.txt

```diff
@@ -60,3 +60,4 @@ tw-text-subhead
 tw-text-base
 tw-text-small
 tw-text-link
+tw-whitespace-normal
```

Comment:
Do we also need `tw-whitespace-pre-wrap` here?

Reply:
I think it does recognize it after all we don't need this